src/graphql/getter.test.ts

```diff
@@ -1091,6 +1091,93 @@ describe('nearImage searchers', () => {
   });
 });
 
+describe('nearMedia searchers', () => {
+  test('a query with a valid nearVideo', () => {
+    const mockClient: any = {
+      query: jest.fn(),
+    };
+
+    const subQuery = `(nearVideo:{video:"iVBORw0KGgoAAAANS"})`;
+    const expectedQuery = `{Get{Person` + subQuery + `{name}}}`;
+
+    new Getter(mockClient)
+      .withClassName('Person')
+      .withFields('name')
+      .withNearVideo({ video: 'iVBORw0KGgoAAAANS' })
+      .do();
+
+    expect(mockClient.query).toHaveBeenCalledWith(expectedQuery);
+  });
+
+  test('a query with a valid nearAudio', () => {
+    const mockClient: any = {
+      query: jest.fn(),
+    };
+
+    const subQuery = `(nearAudio:{audio:"iVBORw0KGgoAAAANS"})`;
+    const expectedQuery = `{Get{Person` + subQuery + `{name}}}`;
+
+    new Getter(mockClient)
+      .withClassName('Person')
+      .withFields('name')
+      .withNearAudio({ audio: 'iVBORw0KGgoAAAANS' })
+      .do();
+
+    expect(mockClient.query).toHaveBeenCalledWith(expectedQuery);
+  });
+
+  test('a query with a valid nearThermal', () => {
+    const mockClient: any = {
+      query: jest.fn(),
+    };
+
+    const subQuery = `(nearThermal:{thermal:"iVBORw0KGgoAAAANS"})`;
+    const expectedQuery = `{Get{Person` + subQuery + `{name}}}`;
+
+    new Getter(mockClient)
+      .withClassName('Person')
+      .withFields('name')
+      .withNearThermal({ thermal: 'iVBORw0KGgoAAAANS' })
+      .do();
+
+    expect(mockClient.query).toHaveBeenCalledWith(expectedQuery);
+  });
+
+  test('a query with a valid nearDepth', () => {
+    const mockClient: any = {
+      query: jest.fn(),
+    };
+
+    const subQuery = `(nearDepth:{depth:"iVBORw0KGgoAAAANS"})`;
+    const expectedQuery = `{Get{Person` + subQuery + `{name}}}`;
+
+    new Getter(mockClient)
+      .withClassName('Person')
+      .withFields('name')
+      .withNearDepth({ depth: 'iVBORw0KGgoAAAANS' })
```

Comment:
you can also a test (I would say that 1 would be enough) that shows that `distance` and `certainty` also is being properly serialized

Reply:
Will do!